src/ontology/cl-edit.owl

```diff
@@ -27663,6 +27673,14 @@ SubClassOf(obo:CL_4023028 ObjectSomeValuesFrom(obo:RO_0002162 obo:NCBITaxon_1009
 SubClassOf(obo:CL_4023028 ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0061534))
 SubClassOf(obo:CL_4023028 ObjectSomeValuesFrom(obo:RO_0002292 obo:PR_P60041))
 
+# Class: obo:CL_4023029 (indirect medium spiny neuron)
```

Comment:
I believe it is more accurate to label this cell type 'indirect pathway medium spiny neuron' since "indirect" describes the pathway in which the neurone is located and not a property of the neurone itself. Additionally 'indirect pathway medium spiny medium neuron' is what is used in the references provided.